cmd/trade.go

```diff
@@ -722,6 +726,26 @@ func runTradeCmd(options inputs) {
 	bot.Start()
 }
 
+func getUserID(l logger.Logger, botConfig trader.BotConfig) (string, error) {
+	var userIDPrehash string
+	if botConfig.IsTradingSdex() {
+		userIDPrehash = botConfig.TradingAccount()
+	} else {
+		exchangeAPIKeys := botConfig.ExchangeAPIKeys.ToExchangeAPIKeys()
+		userIDPrehash = exchangeAPIKeys[0].Key
```

Comment:
can the exchangeAPIKeys be empty here? if so it would result in an error, may be good to check the length and return an error if it is empty